test/uxCapture.test.js

```diff
@@ -247,11 +253,95 @@ describe('UXCapture', () => {
 					.find(mark => mark.name === MOCK_MARK_1_1)
 			).toBeTruthy();
 
+			UXCapture.mark(MOCK_MARK_1_2, false);
+
 			expect(
 				window.performance
 					.getEntriesByType('measure')
 					.find(measure => measure.name === MOCK_MEASURE_1)
 			).toBeTruthy();
 		});
 	});
+
+	describe('startTransition', () => {
+		beforeAll(() => {
+			UXCapture.create({ onMark, onMeasure });
+		});
+
+		beforeEach(() => {
+			ExpectedMark.clearExpectedMarks();
+			onMark.mockClear();
+			onMeasure.mockClear();
+
+			UXCapture.startTransition();
```

Comment:
currently, `startTransition` will call `ExpectedMark.clearExpectedMarks` - is line 272 redundant?